test/PgSubscription.js

```diff
@@ -298,6 +298,16 @@ function(test, done){
 
     Meteor.setTimeout(function() {
       test.equal(players.length, expectedRows.length);
+
+      // Test multiple quick changes to subscription
```

Comment:
Can you make a separate test case? It makes it much easier to debug if there is a basic change case and then a separate "quick change" test case.